db/migrate/20250214015507_create_ticket_comments.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CreateTicketComments < ActiveRecord::Migration[7.2]
+  def change
+    create_table :ticket_comments do |t|
+      t.references :ticket, null: false
+      t.string :comment
```

Comment:
`string` translates to `VARCHAR(255)` I believe, This is most likely too small, you should consider going with `t.text` instead